app/src/main/java/at/aau/edu/wizards/ui/lobby/LobbyViewModel.kt

```diff
@@ -30,17 +30,26 @@ class LobbyViewModel(
         server.stopBroadcasting()
     }
 
+    var numplayer = 1
+    var checktoomanyplayer = false
```

Comment:
```
var propertyName: String = "abc"
    private set // the setter is private and has the default implementation
```

You should use something like that for both properties. It's basically equivalent to following java code

```
private boolean checkTooManyPlayer = false;

public boolean getCheckTooManyPlayer() {
    return checkTooManyPlayer;
}
```

Reply:
completed